lib/node_modules/@stdlib/blas/ext/base/gnansumkbn2/benchmark/benchmark.js

```diff
@@ -38,17 +53,7 @@ var gnansumkbn2 = require( './../lib/main.js' );
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x;
-	var i;
-
-	x = [];
-	for ( i = 0; i < len; i++ ) {
-		if ( randu() < 0.2 ) {
-			x.push( NaN );
-		} else {
-			x.push( ( randu()*10.0 ) - 20.0 );
-		}
-	}
+	var x = filledarrayBy( len, 'float64', rand );
```

Comment:
Let's keep it a plain JS array here:

```suggestion
	var x = filledarrayBy( len, 'generic', rand );
```